x-pack/plugins/security_solution/cypress/cypress.json

```diff
@@ -11,8 +11,8 @@
   "trashAssetsBeforeRuns": false,
   "video": false,
   "videosFolder": "../../../target/kibana-security-solution/cypress/videos",
-  "viewportHeight": 900,
-  "viewportWidth": 1440,
+  "viewportHeight": 946,
+  "viewportWidth": 1680,
```

Comment:
I'm curious as to why this was updated

Reply:
I updated the values with the ones I have on my browser for the screen size of my laptop, thinking it would be more realistic, but I may be wrong.